lib/va1010_forms/enrollment_system/service.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require 'benchmark'
+require 'common/client/base'
+require 'hca/configuration'
+require 'hca/overrides_parser'
+
+module VA1010Forms
+  module EnrollmentSystem
+    class Service < Common::Client::Base
+      include ActionView::Helpers::NumberHelper
+
+      configuration HCA::Configuration
+
+      # @param [Hash] user_identifier
+      # @example { 'icn' => user.icn, 'edipi' => user.edipi }
+      def initialize(user_identifier = nil)
+        super()
+        @user_identifier = user_identifier
+      end
+
+      def submit(parsed_form, form_id)
+        formatted = HCA::EnrollmentSystem.veteran_to_save_submit_form(
+          parsed_form,
+          @user_identifier,
+          form_id
+        )
+        submission_body = submission_body(formatted)
+        response = perform(:post, '', submission_body)
+
+        root = response.body.locate('S:Envelope/S:Body/submitFormResponse').first
+        form_submission_id = root.locate('formSubmissionId').first.text.to_i
+
+        {
+          success: true,
+          formSubmissionId: form_submission_id,
+          timestamp: root.locate('timeStamp').first&.text || Time.now.getlocal.to_s
+        }
+      rescue => e
+        Rails.logger.error "#{form_id} form submission failed: #{e.message}"
+        raise e
```

Comment:
@LindseySaari I added a small rescue block that logs the error message and raises the error. I updated the corresponding test as well.